x/airdrop/keeper/proposal_handler.go

```diff
@@ -49,6 +49,12 @@ func HandleRegisterZoneDropProposal(ctx sdk.Context, k Keeper, p *types.Register
 			return fmt.Errorf("claim record %d, %w", i, err)
 		}
 
+		for j, ca := range cr.ActionsCompleted {
```

Comment:
Should this just be in cr.ActionsCompleted != nil - it should be set here at all right?